pkgs/build-support/build-gog-package/gogextract.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, python3, fetchFromGitHub }:
```

Comment:
I think this should be under tools/misc or similar. You can certainly use the program outside of buildGOGPackage.